lexicon/modification.py

```diff
@@ -0,0 +1,156 @@
+__all__ = ["StaticLexiconJob", "MergeLexiconJob"]
+
+from collections import OrderedDict
+
+from sisyphus import Job, Task
+
+from i6_core.lib import lexicon
+from i6_core.util import write_xml
+
+
+class StaticLexiconJob(Job):
+    """
+    Create a bliss lexicon file from a static Lexicon.
+
+    Supports optional sorting of phonemes and lemmas.
+
+    Example for a static lexicon:
+
+    .. code: python
+
+        static_lexicon = lexicon.Lexicon()
+        static_lexicon.add_lemma(
+            static_lexiconicon.Lemma(
+                orth=["[SILENCE]", ""],
+                phon=["[SILENCE]"],
+                synt=[""],
+                special="silence",
+                eval=[""],
+            )
+        )
+        static_lexicon.add_lemma(
+            static_lexiconicon.Lemma(
+                orth=["[UNKNOWN]"],
+                phon=["[UNKNOWN]"],
+                synt=[["<UNK>"]],
+                special="unknown",
+            )
+        )
+        static_lexicon.add_phoneme("[SILENCE]", variation="none")
+        static_lexicon.add_phoneme("[UNKNOWN]", variation="none")
+    """
+
+    def __init__(self, static_lexicon, sort_phonemes, sort_lemmas, compressed=False):
```

Comment:
here sorting of phonemes and lemmas definitely should have default values (False) and I'd set compressed's default to True.